src/typings/block-type.ts

```diff
@@ -41,9 +41,7 @@ enum BlockType {
     /**
      * Lambda reporter with callable function.
```

Comment:
this is a private type from my own scratch mod. I recommend removing it.
sorry for causing this trouble...

Reply:
No, it is necessary. See `make-ctx`.ts.